.github/workflows/release-continuous.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Any Commit
+on: [push, pull_request]
```

Comment:
Does this mean any commit outside of main or a PR also trigger a release?

Reply:
Yeah. I think the code is not frequently update before. I updated the condition, here only push main branch and labeled `trigger: preview` pr.

Reply:
Could you add a `trigger: preview` label?